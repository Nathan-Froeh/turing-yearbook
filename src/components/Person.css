* {
  box-sizing: border-box;
}

article {
  height: 350px;
  width: 260px;
  background: white;
  border-radius: 20px;
  margin: 5px;
  padding: 10px;
}

img {
  height: 230px;
  width: 190px;
  object-fit: cover;
  border-radius: 20px;
}
