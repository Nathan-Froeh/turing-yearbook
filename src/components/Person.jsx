import React from 'react';
import './Person.css';

// PERSON COMPONENT CODE GOES HERE

function Person(props) {

  return(
    <article>
      <img src={props.person.photo} alt={'Photo of ' + props.name}/>
      <h2>{props.person.name}</h2>
      <p>{props.person.quote}</p>
      <h3>{props.person.superlative}</h3>
    </article>
  )
}

export default Person;

