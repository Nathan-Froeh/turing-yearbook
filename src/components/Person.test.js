import React from 'react';
import Person from './Person';
import { shallow } from 'enzyme';


describe('Person', () => {

  it('should render all components in order', () => {
    const components = shallow(
      <Person 
      person={{name:'Nathan', quote:'', superlative:'', photo:''}} 
      key={1}/>
    )
    expect(components).toMatchSnapshot()
  })

})