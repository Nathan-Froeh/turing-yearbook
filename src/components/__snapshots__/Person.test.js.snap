// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Person should render all components in order 1`] = `ShallowWrapper {}`;
